test-configs.yaml

```diff
@@ -485,11 +485,41 @@ device_types:
     filters:
       - blacklist: *allmodconfig_filter
 
+  fsl-ls1012a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
+  fsl-ls1028a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
+
+  fsl-ls1043a-rdb:
```

Comment:
Tested OK on staging with baseline:
https://staging.kernelci.org/test/plan/id/5e9c837d7ce11ab3e8c3c541/

Device type upstream in LAVA:
https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/fsl-ls1043a-rdb.jinja2